account_claims.go

```diff
@@ -53,11 +54,12 @@ func (o *OperatorLimits) Validate(vr *ValidationResults) {
 
 // Account holds account specific claims data
 type Account struct {
-	Imports     Imports        `json:"imports,omitempty"`
-	Exports     Exports        `json:"exports,omitempty"`
-	Identities  []Identity     `json:"identity,omitempty"`
-	Limits      OperatorLimits `json:"limits,omitempty"`
-	SigningKeys StringList     `json:"signing_keys,omitempty"`
+	Imports         Imports        `json:"imports,omitempty"`
+	Exports         Exports        `json:"exports,omitempty"`
+	Identities      []Identity     `json:"identity,omitempty"`
+	Limits          OperatorLimits `json:"limits,omitempty"`
+	SigningKeys     StringList     `json:"signing_keys,omitempty"`
+	UserRevocations RevocationList `json:"user_revocations,omitempty"`
```

Comment:
We need user_ in the json name? Its scoped here so I think revocations may be ok.

Reply:
done in next update